spec/features/work_packages/details/inplace_editor/subject_editor_spec.rb

```diff
@@ -11,6 +11,7 @@
   let(:field) { WorkPackageField.new page, property_name }
 
   before do
+    maximize(page)
     allow(User).to receive(:current).and_return(user)
     visit project_work_packages_path(project)
```

Comment:
There are other specs as well (description_editor, work_package_field) that should suffer from the same problems. How about we extract this to a shared context (in shared_examples.rb) and reuse it in all the places?